cmd/agent/gui/views/templates/generalStatus.tmpl

```diff
@@ -184,7 +184,14 @@
                 {{ $check -}}<br>
                 {{- range $instance := $instances }}
                   {{- range $k,$v := $instance }}
-                      {{ $k }} : {{ $v }}<br>
+                    - instance_name: {{ .instance_name }}<br>
+                      instance_bean_count: {{ .instance_bean_count }}<br>
```

Comment:
For the three `instance_*_count` fields, lets put them at the bottom of the list and group them under a nested title "Internal JMXFetch telemetry" or something along these lines.

These are not the most intuitive fields so I don't want to highlight them too much on the status page. They're mostly intended to have extra debugging information in support cases, not for day-to-day use by customers. 

We may also want to move them to the `-v` verbose status page. @rayz can you look into moving these to the verbose status page? See @gh123man 's original PR to add verbose status page here https://github.com/DataDog/datadog-agent/pull/16814

specifically [this commit](https://github.com/DataDog/datadog-agent/commit/fe9aa2c9b01e5782d1c54ddb31bce7be97242e0e) is doing the "plumbing" of implementing "verbose"

Reply:
➕ You will just need to pass in the verbose flag as a template variable (logs agent does things a little differently).